src/components/Tooltip/HoverableTooltip/index.tsx

```diff
@@ -0,0 +1,125 @@
+import {BoundsObserver} from '@react-ng/bounds-observer';
+import type {ForwardedRef} from 'react';
+import React, {forwardRef, memo, useCallback, useRef} from 'react';
+import Hoverable from '@components/Hoverable';
+import BaseTooltip from '@components/Tooltip/BaseTooltip';
+import type {HoverableTooltipProps, TooltipRect} from '@components/Tooltip/types';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+
+const hasHoverSupport = DeviceCapabilities.hasHoverSupport();
+
+/**
+ * A component used to wrap an element intended for displaying a tooltip. The term "tooltip's target" refers to the
+ * wrapped element, which, upon hover, triggers the tooltip to be shown.
+ * @param {propTypes} props
+ * @returns {ReactNodeLike}
+ */
```

Comment:
Comment at the wrong place! Regarding the word-to-word specific comment content, we can come back to it later.